presto-docs/src/main/sphinx/release/release-0.206.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.206
+=============
+
+General Changes
+---------------
+
+* Fix execution failure for certain queries containing a join followed by an aggregation
+  when ``dictionary_aggregation`` is enabled.
+* Fix planning failure when a query contains a ``GROUP BY``, but the cardinality of the
+  grouping columns is 1, e.g., ``SELECT c1, sum(c2) FROM t WHERE c1 = 'foo' GROUP BY c1``.
+* Fix high memory pressure on the coordinator during the execution of queries
+  using bucketed execution.
+* Add new semantics for date/time types. The new semantics conforms to the SQL spec.
+  It affects the ``TIMESTAMP`` (aka ``TIMESTAMP WITHOUT TIME ZONE``) type,
+  ``TIME`` (aka ``TIME WITHOUT TIME ZONE``) type, and ``TIME WITH TIME ZONE`` type.
+
+.. note::
+
+  The legacy behavior remains default.
+  At this time, it is not recommended to turn on the new semantics.
+  For any connector that supports date/time types, code change is required before the connector
+  can work correctly with the new semantics. None of the bundled connectors have been updated yet.
+  In addition, the new semantics is also not stable yet as more breaking changes are planned.
+
+* Add :func:`ST_Union`, :func:`ST_Geometries`, :func:`ST_PointN`, :func:`ST_InteriorRings`,
+  and :func:`ST_InteriorRingN` geospatial functions.
+* Add :func:`split_to_multimap` function.
+* Expand :func:`approx_distinct` function to support the following types:
+  ``INTEGER``, ``SMALLINT``, ``TINYINT``, ``DECIMAL``, ``REAL``, ``DATE``,
+  ``TIMESTAMP``, ``TIMESTAMP WITH TIME ZONE``, ``TIME``, ``TIME WITH TIME ZONE``, ``IPADDRESS``.
+* Add a resource group ID column to the ``system.runtime.queries`` table.
+* Add support for executing ``ORDER BY`` without ``LIMIT`` in a distributed manner.
+* Add implicit coercion from ``VARCHAR(n)`` to ``CHAR(n)``, and remove implicit coercion the other way around.
+  As a result, comparing a ``CHAR`` with a ``VARCHAR`` will now follow
+  trailing space insensitive ``CHAR`` comparison semantics.
+* Fix average row size computation to account for non-null rows only during query cost estimation.
```

Comment:
"for non-null rows only during ... estimation"

The phrasing is ambiguous. Is this about "non-null rows only" or "only during ... estimation"?